### PR TITLE
data: Add linkefree profile (Adetoye)

### PR DESCRIPTION
**Pull request by adetoye-dev**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ x] My change requires changes to the documentation.
- [ x] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2890"></a>

### CODE REVIEW

File: data/adetoye-dev.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/adetoye-dev",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/adetoye-dev",
+      "icon": "FaGithub"
+    },
```
Comment (Aadarsh805) [inline data/adetoye-dev.json:27]: this comma at the end of line 27 needs to be removed

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Comment by adetoye-dev** [comment]
Thanks for reviewing my PR @Aadarsh805 .
I have made the changes

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Comment by adetoye-dev** [comment]
Thanks Pradumna 😇👏

**Pull request merged**